react/features/base/ui/components/JitsiThemeProvider.web.js

```diff
@@ -38,9 +45,10 @@ function JitsiThemeProvider(props: Props) {
  */
 function _mapStateToProps(state) {
     const { muiBrandedTheme } = state['features/dynamic-branding'];
+    const { direction } = state['features/base/ui'];
 
     return {
-        _theme: muiBrandedTheme || BaseTheme
+        _theme: muiBrandedTheme || direction === 'rtl' ? rltTheme : ltrTheme
```

Comment:
Doesn't this mean the direction won't work when dynamic branding is used? Maybe we could inject it, if missing?

Reply:
I stored the `muiBrandedRtlTheme` state in the store, when there is a `customTheme`. But I have not tested it.